src/main/java/org/fcrepo/auth/webac/LinkHeaderProvider.java

```diff
@@ -71,7 +71,7 @@
                 new DefaultIdentifierTranslator(internalSession);
         final ListMultimap<String, String> headers = ArrayListMultimap.create();
 
-        LOGGER.debug("Adding WebAC Link Header for Resource: {}", resource);
+        LOGGER.debug("Adding WebAC Link Header for Resource: {}", resource.getPath());
```

Comment:
Yeah, that's what I meant, the semantics are what we take over. I agree that `getIdentifier` is much more in line with the thrust of the public API.